packages/e2e/features/ui/components/storage/storage-browser/insufficient-permissions.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Storage Browser denied access without sufficient permissions
+
+  Background:
+    Given I'm running the example "ui/components/storage/storage-browser/default-auth"
+
+  @react
+  Scenario: Attempt to access forbidden folder
+    When I type my "email" with status "CONFIRMED"
+    Then I type my password
+    Then I see the "Sign in" button
+    When I click the "Sign in" button
+    When I click the button containing "forbidden"
+    Then I don't see "amplify-table"
```

Comment:
This seems pretty implementation specific. Can we check for a forbidden error or something?

Reply:
Updated to watch for a 403